macros/models/_sector/tokens/transfers_base.sql

```diff
@@ -25,6 +25,7 @@ WITH transfers AS (
         {% if is_incremental() %}
         AND {{incremental_predicate('block_time')}}
         {% endif %}
+        and block_time >= date '2024-01-01'
```

Comment:
hardcoded for historical to be quicker
this would make joins to itself quicker since target is smaller, but same would apply to all strategies at least